config/routes.rb

```diff
@@ -310,6 +310,9 @@
     end
     resources :ssh_keys, :only => [:new, :create, :destroy]
   end
+
+  post 'users/jwt', to: 'jwt#user_jwt'
```

Comment:
you can include it in the users routing block
```suggestion
  
resources :users, :except => [:show] do
    ...
    resources :ssh_keys, :only => [:new, :create, :destroy]
    resources :jwt, :only => [:create]
  end



```

Reply:
But this will generate `/users/:user_id/jwt` route,
but I need `/users/jwt`, so the only solution for me is:
```
  namespace :users do
    post 'jwt', to: 'jwt#create'
  end
```

Reply:
shouldnt you need the token per user? not sure I follow?

in term of reset, the correct url should be /users/id/jwt AFAIU

Reply:
Well I don't need `:id` at all, because I generate token for the current user
```
    def create
      render json: { jwt: User.current.jwt_token!(expiration: expiration) }
    end
```